### ISSUE TITLE
Number input for the time for data saving

### ISSUE THREAD

**Issue by patrickrobrecht** [pluginkollektiv/statify#42]
A input field instead of the selection field would be much more flexible.

**Comment by websupporter** [comment]
Maybe we could use `0` as an indicator for "do not delete"?

**Comment by bueltge** [comment]
But is this flexibility really necessary, have we request from this or it is only a wish? At the first usage of the plugin I thought also that an flexible value is great, but after this, I think is not necessary and this solid values are enough and not to much data in the database. If we change this, we should also check for a value border, so that the performance it works.

**Comment by websupporter** [comment]
Well, some days ago I started to use the Extended Plugin, which comes with some great enhancements. I talked with @patrickrobrecht if his plugin steps here into it in order to save the data longer (for some nice overviews like month to month and stuff). It doesn't and the maximum of the select is 365 days. In this discussion the idea came up to remove the select and use an input-field.

So broadly speaking the flexibility becomes necessary, when we think of Statify as the data generator. I am not so happy with the input UX wise, but I think a more flexible approach would be nice.
